test/hotspot/gtest/x86/test_assemblerx86.cpp

```diff
@@ -67,6 +67,7 @@ static void asm_check(const uint8_t *insns, const uint8_t *insns1, const unsigne
 
 TEST_VM(AssemblerX86, validate) {
   FlagSetting flag_change_apx(UseAPX, true);
+  VM_Version::set_bmi1_cpuFeatures();
```

Comment:
Let us also call the set_avx_cpuFeatures() and set_evex_cpuFeatures() in TEST_VM.

Reply:
Thanks, done.